diff -up openal-soft-openal-soft-1.20.1/CMakeLists.txt.omv~ openal-soft-openal-soft-1.20.1/CMakeLists.txt
--- openal-soft-openal-soft-1.20.1/CMakeLists.txt.omv~	2020-06-10 01:01:34.234933571 +0200
+++ openal-soft-openal-soft-1.20.1/CMakeLists.txt	2020-06-10 01:02:33.502808457 +0200
@@ -281,7 +281,7 @@ ELSE()
     SET(CMAKE_REQUIRED_FLAGS "${OLD_REQUIRED_FLAGS} -Wattributes -Werror")
 
     CHECK_C_SOURCE_COMPILES("int foo() __attribute__((visibility(\"protected\")));
-                             int main() {return 0;}" HAVE_GCC_PROTECTED_VISIBILITY)
+                             int main() {return 0;} Let's barf here to fix things" HAVE_GCC_PROTECTED_VISIBILITY)
     IF(HAVE_GCC_PROTECTED_VISIBILITY)
         SET(EXPORT_DECL "__attribute__((visibility(\"protected\")))")
     ELSE()
