diff -up openal-soft-1.21.1/examples/alffplay.cpp.omv~ openal-soft-1.21.1/examples/alffplay.cpp
--- openal-soft-1.21.1/examples/alffplay.cpp.omv~	2022-01-16 00:10:20.115798762 +0100
+++ openal-soft-1.21.1/examples/alffplay.cpp	2022-01-16 00:10:23.633811102 +0100
@@ -1713,7 +1713,7 @@ int MovieState::streamComponentOpen(unsi
     if(avcodec_parameters_to_context(avctx.get(), mFormatCtx->streams[stream_index]->codecpar))
         return -1;
 
-    AVCodec *codec{avcodec_find_decoder(avctx->codec_id)};
+    const AVCodec *codec{avcodec_find_decoder(avctx->codec_id)};
     if(!codec || avcodec_open2(avctx.get(), codec, nullptr) < 0)
     {
         std::cerr<< "Unsupported codec: "<<avcodec_get_name(avctx->codec_id)
